<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Speed Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="overallCpuFrequencyLabel">
      <property name="text">
       <string>Overall CPU Frequency: [placeholder_freq]</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cpuGovernorLabel">
        <property name="text">
         <string>CPU Governor/Frequency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cpuGovernorComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="cpuCoresLabel">
      <property name="text">
       <string>CPU Cores:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="cpuCoresListWidget"/>
    </item>
    <item>
     <widget class="QPushButton" name="applyChangesButton">
      <property name="text">
       <string>Apply Changes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
